### DISCUSSION TITLE
Open3D | BPA on a .obj file is running forever when increasing the radius

### DISCUSSION THREAD

**Discussion by xraise17** [isl-org/Open3D#6357] [General]
Hey everyone, I have a question and I would be very thankful if you could provide me with an answer or even better with resources for me to understand why this is happening. I want to create a mesh using the ball pivoting algorithm (BPA) provided by the Open3D. When running the BPA algorithm with a small radius e.g. 0.05 it completes and the mesh is created. When running it with a bigger radius e.g. 0.5 the script doesn't finish. Why is this happening? The object is a tree without leafs just the trunk and branches.

**Comment by germanros1987** [comment]
Could you please provide an example?

**Reply by xraise17** [reply]
Hey sorry for the late response, down sampling the point cloud of the scene made it finish.
